### PR TITLE
feat: pass down raw document to decentralized renderer

### PR DESCRIPTION
**Pull request by Nebulis**
Provide raw document to decentralized renderer.

We need it for acra so that acra renderer can perform extra check regarding the target hash of the document

### CODE REVIEW

File: src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ class DecentralisedRenderer extends Component {
   async renderDocument(certificate) {
     const { childFrameConnection } = this.state;
     const child = await childFrameConnection;
-    await child.renderDocument(certificate);
+    await child.renderDocument(certificate, this.props.certificate);
```
Comment (yehjxraymond) [inline src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js]: Might be unclear to new dev what's the difference. Maybe can rename this.props.certificate? 
```
const rawDocument = this.props.certificate
```

Comment (Nebulis) [reply src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js]: actually I dunno the diff between document and certificate in props ...

Comment (Nebulis) [reply src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js]: done

File: src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js [hunk 2]
```diff
@@ -80,7 +82,10 @@ class DecentralisedRenderer extends Component {
     this.setState({ childFrameConnection });
 
     childFrameConnection.then(frame =>
-      frame.renderDocument(getData(this.props.certificate))
+      frame.renderDocument(
+        getData(this.props.certificate),
```
Comment (yehjxraymond) [inline src/components/DecentralisedTemplateRenderer/DecentralisedRenderer.js]:
```suggestion
        getData(rawDocument),
```

**Pull request merged**